app/assets/javascripts/ui_models/editor.ts

```diff
@@ -94,11 +93,20 @@ export class Editor {
     this._onNoteValueChange = callback;
   }
 
+  private setTemplateNote(note: SNNote) {
+    this.isTemplateNote = true;
+    this.note = note;
+    if (this._onNoteChange) {
+      this._onNoteChange();
+    }
+  }
+
   /**
    * Sets the editor contents by setting its note.
    */
   public setNote(note: SNNote) {
```

Comment:
What about just passing a boolean parameter `isTemplate` to this function instead? Because `setTemplateNote` is only called once so doesn't really make sense to be it's own function, and it's duplicating 90% of `setNote`.